Source/CutomePlayer/YoutubeVideoPlayer.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  YoutubeVideoPlayer.swift
+//  edX
+//
+//  Created by Andrey on 9/4/18.
+//  Copyright © 2018 edX. All rights reserved.
+//
+
+class YoutubeVideoPlayer: VideoPlayer{
+
+    let playerView: YTPlayerView
+    var videoId: String
+
+    override var currentTime: TimeInterval {
+        return Double(playerView.currentTime())
+    }
+    override init(environment : Environment) {
+        playerView = YTPlayerView()
+        videoId = String()
+        super.init(environment: environment)
+        playerView.delegate = self
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .black
+        createYoutubePlayer()
+    }
+
+    private func createYoutubePlayer() {
+        videoPlayerProtraitView(portraitView: UIDevice.current.orientation.isPortrait)
+        view.addSubview(playerView)
+        UINavigationBar.appearance().barTintColor = .black
+    }
+
+    func videoPlayerProtraitView(portraitView: Bool){
+        let screenSize: CGRect = UIScreen.main.bounds
+
+        if portraitView{
+            playerView.frame = CGRect(x: 0, y: 0, width: screenSize.width, height: screenSize.height * 0.3)
```

Comment:
Is this 0.3 here right? Does it work on all screensizes? Or should it be calculated using 16:9 from the width?